ote/resources/public/css/styles.css

```diff
@@ -21,6 +21,7 @@ body {
 h1, h2, h3, h4, h5, h6,
 .h1, .h2, .h3, .h4, .h5, .h6 {
     margin-bottom: 1rem;
+    font-family: Monserrat, sans-serif;
```

Comment:
You have a typo here. It's "Montserrat"
Also:
`font-family: 'Montserrat', 'montserrat', 'Source Sans', Arial, sans-serif;` would be a good fallback list. Source sans is very similar to montserrat.